### ISSUE TITLE
Expand raft

### ISSUE THREAD

**Issue by Tinchus2009** [slic3r/Slic3r#4454]
_Is this a new feature request?_
Yes

I ahve been useing raft not very oftenn really, but today I found  some issues. On of them it is actually a feature request: the ability to expand the size of the raft. Why? The purpose, one of them, of a raft would be to give more support to object with small bad surface. For example a tall small radious tower. Right now if you create a raft it will have the same size as the object's base. If the object base in contact with the bed has, for example, 1 cms and it is 15 cms tall... we have a problem wich can not be solved with a raft. 
We should have the ability to increase the size of the raft to be expandable X mm bigger than object's base.

**Comment by justUmen** [comment]
Yeah, I have this problem now. I just can't simply increase the size of the raft and the part becomes loose at the end of the print. It is just too weak by default and can't make it bigger...
